server/service/src/sync/program_schemas/patient_ui_schema.json

```diff
@@ -438,19 +438,10 @@
               "label": "Notes",
               "elements": [
                 {
-                  "type": "Control",
+                  "type": "Notes",
                   "label": "Patient Notes",
                   "scope": "#/properties/notes",
                   "options": {
-                    "detail": {
-                      "type": "VerticalLayout",
-                      "elements": [
-                        {
-                          "type": "Note",
-                          "scope": "/properties"
-                        }
-                      ]
-                    },
```

Comment:
As suggested, the "Notes" component should have to define its children (since they'll always be "Note" elements). See my slightly hacky solution above for -- I've basically just hard-coded this sub-schema and injected it into options.